### ISSUE TITLE
Parsing issue in tag 08D0

### ISSUE THREAD

**Issue by pbelmans** [gerby-project/plastex#20]
It looks like plasTeX is spewing out Unicode things again, not sure what triggered it this time.

**Comment by pbelmans** [comment]
Likewise on 03V7.

**Comment by chngr** [comment]
I don't see these problems either on my build or Mercer. Can you drop the offending LaTeX code here?

**Comment by pbelmans** [comment]
Yes, I agree there is probably something wrong with versions on my side. Tomorrow I'll start from a clean slate.

**Comment by pbelmans** [comment]
Following [my own instructions](http://pbelmans.ncag.info/blog/2017/08/27/introduction-gerby-plastex/), with the version of plasTeX currently in the repository, I still get these errors. There must be something off with my configuration though, but I can't figure out what...

So on tag 03V7 the issue is the second equation, i.e. the one after "Applying Theorem [...]". The determinant is dropped from the Unicode output, and so is the entire second line (except for the equals sign and the dot.

But the reason why this showed up in the headless browser test is because in the paragraph labelled **Projective case** there is a `Misplaced &` MathJax error.

**Comment by chngr** [comment]
I see! In your instructions, you do not include `\usepackage{amsmath}` in the preamble! This is probably the root of many of these problems. For example, the `\begin{matrix} ... \end{matrix}` environment is defined in `amsmath`, so `plasTeX` is confused without it. (I think the reason why it has the strange behaviour is because `matrix` does mean something even in `LaTeX`, but it's not what you expect.)

My previous runs have been based on running `plasTeX` in which I followed your instructions in modifying `book.tex`, PLUS including `\usepackage{amsmath}` in the preamble. In those cases, there were no errors. I have recreated my database tonight without the `amsmath` package and have been able to replicate your errors (and many more!).
